tox.ini

```diff
@@ -25,15 +26,20 @@ commands =
     python setup.py check -m -r -s
 
 [testenv:pep8]
-basepython = python2.7
-deps = flake8==2.3.0
+basepython = python2
+deps = flake8==3.3.0
```

Comment:
Upgrade the version of flake8 being used since the version being used was around 2 years old.